darktrace/plugin.spec.yaml

```
@@ -0,0 +1,99 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: darktrace
+title: DarkTrace
+version: 1.0.0
+description: This plugin utilizes DarkTrace to  detects novel attacks and insider threats at an early stage
+vendor: rapid7
+support: rapid7
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/darktrace
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.darktrace.com/
+tags:
+- darktrace
+- immune
+- system
+hub_tags:
+  use_cases: [threat_detection_and_response]
+  keywords: [darktrace, immune, system, detects]
+  features: []
+connection:
+  url:
+    title: URL
+    type: string
+    description: Connection URL
```

```suggestion
    description: Connection URL
```

Should be API URL if it's an API URL or Instance URL if it's an instance URL